src/test/java/bio/terra/service/snapshotbuilder/query/QueryTest.java

```diff
@@ -70,96 +68,11 @@ void renderSqlGroupBy(SqlRenderContext context) {
     FieldPointer fieldPointer = new FieldPointer(tablePointer, "field");
     FieldVariable fieldVariable = new FieldVariable(fieldPointer, sourceVariable);
     Query query =
-        new Query(List.of(fieldVariable), List.of(sourceVariable), List.of(fieldVariable));
+        new Query.Builder()
+            .select(List.of(fieldVariable))
+            .tables(List.of(new Table(sourceVariable)))
+            .groupBy(List.of(fieldVariable))
+            .build();
     assertThat(query.renderSQL(context), is("SELECT t.field FROM table AS t GROUP BY t.field"));
   }
-
-  @ParameterizedTest
-  @ArgumentsSource(SqlRenderContextProvider.class)
-  void renderComplexSQL(SqlRenderContext context) {
-    TablePointer tablePointer = TablePointer.fromTableName(Person.TABLE_NAME);
```

Comment:
why do we no longer need this test?

Reply:
This doesn't add any test coverage, everything it's testing has already been tested.